packages/facade/src/apis/sheets/__tests__/f-range.spec.ts

```diff
@@ -809,4 +809,14 @@ describe('Test FRange', () => {
             expect.arrayContaining([1, 2, 3, 4, null, null, null, null, null, null]),
         ]));
     });
+
+    it('Range setNumberFormat', () => {
+        univerAPI.getHooks().onRendered(() => {
+            const activeSheet = univerAPI.getActiveWorkbook()?.getActiveSheet();
```

Comment:
In unit test, you can just use assetion systax. 
```suggestion
            const activeSheet = univerAPI.getActiveWorkbook()!.getActiveSheet();
```